Formula/qgis3.rb

```diff
@@ -350,6 +358,27 @@ def pour_bottle?
   def install
     ENV.cxx11
 
+    printf "\n\033[31mIn case you have installed another version of Python (Anaconda, Miniconda or if you used the installer provided by python.org).\e[0m\n\n"
+
+    printf "\e[0mIf the installation failed due to the problem reported in \e[32mhttps://github.com/OSGeo/homebrew-osgeo4mac/issues/520\e[0m\n\n"
+
+    printf "1 - Try after doing:\n\n"
+
+    printf "    $ brew unlink python && brew link --force python\n\n"
+
+    printf "    $PATH (Check that there is no other version)\n\n"
+
+    printf "2 - As a last resort, remove all your Python installations. Clean your PATH, install Python through Homebrew and (1).\n\n"
```

Comment:
I'm not sure recommending removal of other python installs is a good idea. Lots of people have very specific installations and packages and removing them seems like it could create more issues than it solves.

Reply:
I just left the first recommendation